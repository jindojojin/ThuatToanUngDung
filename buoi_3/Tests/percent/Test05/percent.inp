70
2 18 5 1 74
10 25 19 19 27
3 15 6 6 70
2 28 2 12 56
6 43 28 17 6
2 5 3 14 76
3 33 22 17 25
9 46 18 7 20
2 39 10 13 36
8 50 26 15 1
4 12 3 14 67
4 15 22 12 47
4 19 18 5 54
3 8 28 20 41
4 42 20 19 15
7 36 21 3 33
0 40 60 0 0
5 49 27 6 13
1 30 11 11 47
7 2 4 9 78
10 24 25 15 26
7 41 17 17 18
2 9 25 20 44
7 24 18 19 32
9 33 10 2 46
4 16 20 19 41
5 31 28 7 29
4 37 12 6 41
5 23 1 10 61
8 24 28 13 27
7 41 12 17 23
6 18 16 15 45
2 3 21 11 63
2 25 17 11 45
8 42 8 18 24
8 38 4 4 46
6 10 20 19 45
2 39 3 7 49
7 31 4 9 49
1 42 13 16 28
25 0 25 50 0
9 34 26 2 29
3 1 2 17 77
5 21 7 2 65
9 50 19 5 17
2 35 24 20 19
9 39 1 9 42
8 18 9 14 51
9 34 18 2 37
1 18 14 15 52
10 17 6 12 55
1 50 20 17 12
9 4 15 9 63
5 10 20 3 62
6 36 4 4 50
4 38 5 4 49
9 1 19 19 52
1 47 9 2 41
10 49 20 18 3
3 23 29 13 32
9 30 11 18 32
9 42 16 9 24
7 12 3 15 63
3 6 19 7 65
3 37 6 14 40
10 43 25 17 5
2 49 3 12 34
2 50 18 17 13
3 40 6 13 38
1 11 4 10 74
