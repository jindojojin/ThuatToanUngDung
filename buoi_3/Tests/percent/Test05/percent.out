100
100
100
50
100
100
100
100
100
100
100
100
100
100
100
100
5
100
100
100
100
100
100
100
100
100
100
100
100
100
100
100
100
100
50
50
100
100
100
100
4
100
100
100
100
100
100
100
100
100
100
100
100
100
50
100
100
100
100
100
100
100
100
100
100
100
100
100
100
100
